### ISSUE TITLE
Remove uses-permission target (Android manifest)

### ISSUE THREAD

**Issue by denisogolubev** [ionic-team/trapeze#73]
Hello, 

I'm using Project API.
I want to remove all autogenerated uses-permission tags from AndroidManifest.xml after the `npx cap add android` command.
Or I don't understand how to do this or it's not implemented right now.

Can you help me with an answer?
Thanks.

**Comment by mlynch** [comment]
Okay so we don't support deletion in XML at the moment but we should, that would be a very useful thing to have. I've put this on the backlog and will update you here once it lands.

**Comment by denisogolubev** [comment]
And removing attributes will be nice to have too.
Because some attributes are needed only in livereload mode, but in release mode they are insecure and we need to remove them before.

Thank you for the feedback.
Have a great day!

**Comment by mlynch** [comment]
Added support for deleting nodes and attributes. Will be out in the next release. Thanks for the suggestion and feedback!
